src/common/java/net/minecraftforge/gradle/common/util/EclipseHacks.java

```diff
@@ -20,89 +20,58 @@
 
 package net.minecraftforge.gradle.common.util;
 
-import groovy.util.Node;
-import groovy.util.XmlParser;
-import groovy.xml.XmlUtil;
 import net.minecraftforge.gradle.common.task.DownloadAssets;
 import net.minecraftforge.gradle.common.task.ExtractNatives;
-import org.apache.commons.io.IOUtils;
 import org.gradle.api.Project;
 import org.gradle.api.Task;
 import org.gradle.api.tasks.TaskProvider;
+import org.gradle.plugins.ide.api.XmlFileContentMerger;
 import org.gradle.plugins.ide.eclipse.GenerateEclipseClasspath;
-import org.xml.sax.SAXException;
+import org.gradle.plugins.ide.eclipse.model.Classpath;
+import org.gradle.plugins.ide.eclipse.model.ClasspathEntry;
+import org.gradle.plugins.ide.eclipse.model.EclipseModel;
+import org.gradle.plugins.ide.eclipse.model.SourceFolder;
 
 import javax.annotation.Nonnull;
-import javax.xml.parsers.ParserConfigurationException;
 import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
+import java.util.Iterator;
 import java.util.Set;
 
 public class EclipseHacks {
 
-    @SuppressWarnings({"unchecked", "UnstableApiUsage"})
```

Comment:
Since eclipse does not support `UnstableApiUsage` I'm not sure if it's actually logical to remove this.  Everything is normal API, but I might have missed something.